source/utilities.tex

```diff
@@ -19419,3 +19420,1886 @@
+\begin{itemdecl}
+template<class Out>
+  Out vformat_to(Out out, string_view fmt, format_args_t<Out, char> args);
+template<class Out>
+  Out vformat_to(Out out, wstring_view fmt, format_args_t<Out, wchar_t> args);
+template<class Out>
+  Out vformat_to(Out out, const locale& loc, string_view fmt,
+                 format_args_t<Out, char> args);
+template<class Out>
+  Out vformat_to(Out out, const locale& loc, wstring_view fmt,
+                 format_args_t<Out, wchar_t> args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let \tcode{charT} be \tcode{decltype(fmt)::value_type}.
+
+\pnum
+\constraints
+\tcode{Out} satisfies \tcode{OutputIterator<const charT\&>}.
+
+\pnum
+\expects
+\tcode{Out} models \tcode{OutputIterator<const charT\&>}.
+
+\pnum
+\effects
+Places the character representation of formatting
+the arguments provided by \tcode{args},
+formatted according to the specifications given in \tcode{fmt},
+into the range \range{out}{out + N},
+where \tcode{N} is
+\tcode{formatted_size(fmt, args...)} for the functions without a \tcode{loc} parameter and
+\tcode{formatted_size(loc, fmt, args...)} for the functions with a \tcode{loc} parameter.
+If present, \tcode{loc} is used for locale-specific formatting.
+
+\pnum
+\returns
+\tcode{out + N}.
+
+\pnum
+\throws
+\tcode{format_error} if \tcode{fmt} is not a format string.
+\end{itemdescr}
+
+\indexlibrary{\idxcode{format_to_n}}%
+\begin{itemdecl}
+template<class Out, class... Args>
+  format_to_n_result<Out> format_to_n(Out out, iter_difference_t<Out> n,
+                                      string_view fmt, const Args&... args);
+template<class Out, class... Args>
+  format_to_n_result<Out> format_to_n(Out out, iter_difference_t<Out> n,
+                                      wstring_view fmt, const Args&... args);
+template<class Out, class... Args>
+  format_to_n_result<Out> format_to_n(Out out, iter_difference_t<Out> n,
+                                      const locale& loc, string_view fmt,
+                                      const Args&... args);
+template<class Out, class... Args>
+  format_to_n_result<Out> format_to_n(Out out, iter_difference_t<Out> n,
+                                      const locale& loc, wstring_view fmt,
+                                      const Args&... args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let
+\begin{itemize}
+\item \tcode{charT} be \tcode{decltype(fmt)::value_type},
+\item \tcode{N} be
+\tcode{formatted_size(fmt, args...)} for the functions without a \tcode{loc} parameter and
+\tcode{formatted_size(loc, fmt, args...)} for the functions with a \tcode{loc} parameter, and
+\item \tcode{M} be \tcode{min(max(n, 0), N)}.
+\end{itemize}
+
+\pnum
+\constraints
+\tcode{Out} satisfies \tcode{OutputIterator<const charT\&>}.
+
+\pnum
+\expects
+\tcode{Out} models \tcode{OutputIterator<const charT\&>}, and
+\tcode{formatter<}$\tcode{T}_i$\tcode{, charT>}
+meets the \newoldconcept{Formatter} requirements
+for each $\tcode{T}_i$ in \tcode{Args}.
```

Comment:
True, but not editorial. Please file an LWG issue.

Reply:
I may have been wrong.

The non `v`-prefixed functions are OK since the precondition is subsumed via equivalent to semantics as specified in `make_format_args` and family.

For the `v`-prefixed ones, however, they are as a _Constraints:_ on their call site, as specified in the constructor for `basic_format_arg` taking a `T`, which is required to construct the arguments of type `format_args` and family.

Reply:
@johelegp we're fine here, right?  So this can be resolved?  If not, can you please file an LWG issue for it?

Reply:
I'll have to file a LWG issue.

Reply:
After all, there's no real issue here. You could theoretically dance around with `decltype` to get the types needed to make an expression equivalent to calling `make_format_args` and thus skip its preconditions, but that's not practical.